html/static/js/html.js

```diff
@@ -0,0 +1,89 @@
+/* Javascript for HTMLXBlock. */
+var CUSTOM_FONTS, STANDARD_FONTS, _getFonts;
```

Comment:
[We can use ES6](https://github.com/edx/open-edx-proposals/blob/e6a20a0b3f11f99c6c5d9fb3e537db1ac4062e34/oeps/oep-0011-bp-FED-technology.rst#specification), so there's no need to declare variables up front, and you can declare them using `const`:

```javascript
const CUSTOM_FONTS = "Default='Open Sans', Verdana, Arial, Helvetica, sans-serif;";
// ...
const _getFonts = () => CUSTOM_FONTS + STANDARD_FONTS;
```

Reply:
Hmm actually outside of edx-platform we probably can't use arrow functions without a transpilation step, since we need to support IE11. But we can definitely use `const` and declare variables when they're initialized.